pkgs/top-level/python-packages.nix

```diff
@@ -2089,6 +2089,8 @@ in {
 
   httpserver = callPackage ../development/python-modules/httpserver {};
 
+  bleach1_5 = callPackage ../development/python-modules/bleach/1.5.nix {};
```

Comment:
This is a "tensorflow-tensorboard" dependency. Perhaps we won't need it if we decide to drop tensorboard, see my comment on the PR.